client/src/app/employee/employee-list/employee-list.component.html

```diff
@@ -1,54 +1,22 @@
-<table class="table table-hover">
-  <thead>
-    <tr>
-      <th scope="col">Name</th>
-      <th scope="col">Title</th>
-      <th scope="col">Phone</th>
-      <th scope="col">Employment Started On</th>
-      <th scope="col">Status</th>
-      <th scope="col">Action</th>
-    </tr>
-  </thead>
-  <tbody>
-    <tr
-      [ngClass]="{
-        'table-secondary': employee.status === 'Disabled'
-      }"
-      *ngFor="let employee of employees | async"
-    >
-      <th scope="row"> {{ employee.firstName + ' ' + employee.lastName }} </th>
-      <td>{{ employee.title }}</td>
-      <td>{{ employee.phone }}</td>
-      <td>{{ employee.employmentStartedOn | date }}</td>
-      <td>
+<div class="row" *ngFor="let employee of employees">
```

Comment:
There's an argument to be made for making this a Single-File Component (SFC).

There's 0 business logic, the template is small, the component `@Input()` bindings are minimal.

You could even use the `selected` event emitter directly in the template since there's no logic involved:

```html
(click)="selected.emit(employee)"
```

Reply:
👍